controllers/utils/chaosdaemon/chaosdaemon.go

```diff
@@ -67,7 +67,7 @@ func (b *ChaosDaemonClientBuilder) FindDaemonIP(ctx context.Context, pod *v1.Pod
 
 	daemonIP := findIPOnEndpointSlice(&endpointSliceList, nodeName)
 	if len(daemonIP) == 0 {
-		return "", errors.Errorf("cannot find daemonIP on node %s in endpointSliceList %v", nodeName, endpointSliceList)
+		return "", errors.Errorf("cannot find daemonIP on node %s ", nodeName)
```

Comment:
Please revert this change.

Reply:
Thanks @g1eny0ung . I fixed everything except this one.

In a large k8s cluster, simply outputting this `discoveryv1.EndpointSliceList` would make the log huge. So I removed that output.

WDYT?

Reply:
do not print the detail of endpointslicelist looks OK for me, print the node name could be enough for profiling problems.